Source/input/callback.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <iostream>
+#include <functional>
+#include <map>
+
+#include <GLFW/glfw3.h>
+
+#include "input.h"
+
+
+
+class PlayerCallback final {
+private:
+    input::Token mInputToken;
+    static bool mMoveForward;
```

Comment:
`mMoveForward` should not be `static`. 

_Aside: `m` means "member", it is the wrong prefix for statics, should be `s`._